src/components/Tooltip/Tooltip.tsx

```diff
@@ -459,6 +460,9 @@ const Tooltip = ({
     const elementRefs = new Set(anchorRefs)
 
     anchorsBySelect.forEach((anchor) => {
+      if(disableTooltip?.(anchor)){
+        return;
+      }
```

Comment:
This separation between `anchorSelect` and `anchorId` is something we're planning to improve on for v6 (`anchorId` is already deprecated), but until then, we should probably do the check below as well.

Maybe like this:

```ts
    const anchorById = document.querySelector<HTMLElement>(`[id='${anchorId}']`)
    if (anchorById && !disableTooltip?.(anchorById)) {
      elementRefs.add({ current: anchorById })
    }
```

Reply:
Please add one space between `if` and `(` on line 463